Current master branch doesn't build

I'm at git SHA `aa4c449c16c2594288bf5ec73fb882a36b60b6d5` on the master branch.  If I run `./gradlew build` (after successfully running `./gradlew cloneAndBuildDependencies`) I get the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':framework-test:spotlessJavaCheck'.
> The following files had format violations:
      src/main/java/org/checkerframework/framework/test/CheckerFrameworkWPIPerDirectoryTest.java
          @@ -48,7 +48,7 @@
           ···*·····typechecked·ends·with.·Usually,·this·takes·the·form·"all-systems/ProblematicFile.java".
           ···*/
           ··protected·void·doNotTypecheck(
          -··········@UnderInitialization(CheckerFrameworkPerDirectoryTest.class)·CheckerFrameworkWPIPerDirectoryTest·this,
          +······@UnderInitialization(CheckerFrameworkPerDirectoryTest.class)·CheckerFrameworkWPIPerDirectoryTest·this,
           ······String·endswith)·{
           ····int·removeIndex·=·-1;
           ····for·(int·i·=·0;·i·<·testFiles.size();·i++)·{
  Run './gradlew :framework-test:spotlessApply' to fix these violations.
```

Is this expected?

--- a/build.gradle
+++ b/build.gradle
@@ -156,6 +156,9 @@ allprojects {
   }
 
   spotless {
+    // TODO: Temporarily disable running spotlessCheck with the build task.
+    enforceCheck false
+
     format 'misc', {
       // define the files to apply `misc` to
       target '*.gradle', '*.md', '.gitignore'

--- a/build.gradle
+++ b/build.gradle
@@ -156,7 +156,7 @@ allprojects {
   }
 
   spotless {
-    // TODO: Temporarily disable running spotlessCheck with the build task.
+    // TODO: Remove the line below.
     enforceCheck false
 
     format 'misc', {

